pyproject.toml

```diff
@@ -35,13 +35,13 @@ include = ["gaphor/locale/*/LC_MESSAGES/*"]
 exclude = ["**/tests", "gaphor/conftest.py" ]
 
 [tool.poetry.dependencies]
-python = "^3.9"
+python = "^3.9, <3.11"
```

Comment:
@danyeaw  Why the `<3.11` limitation?

Reply:
@amolenaar PyInstaller has a limitation that it can only be used on greater than 3.6, but less than 3.11. If we don't put the same limitation on the Python version to exclude Python 3.11, the dependency resolver in Poetry won't all the version to be updated.